Client/Ecosia/Analytics/Analytics+Configuration.swift

```diff
@@ -73,7 +73,7 @@ extension Analytics {
             defaults.set(false, forKey: identifier)
             return true
         }()
-        return isFirstTime
+        return isFirstTime && EcosiaInstallType.get() != .upgrade
```

Comment:
Nice change! I'm still puzzled how we ensure that this code only applies to the install event. I see it being called from Line 60 in this file, but it's not filtered by any schema like we see in the `appResumeDailyTrackingPluginConfiguration`. Am I missing something how these plugins work?

Reply:
Thanks for your input! It does indeed filter for the schema, and specifically for the `ios_install_event` one.
```swift
        return plugin.filter(schemas: [
            Self.installSchema
        ]) { _ in
            return Self.isFirstInstall(for: identifier)
        }
```
where `installSchema` being: `"iglu:org.ecosia/ios_install_event/jsonschema/1-0-0"`

Reply:
Ah so true! Now I see it.. was hidden in Swift's syntactic sugar 🍬 🙈